card_db_src/translation.md

```diff
@@ -0,0 +1,180 @@
+- The key word `untranslated` MUST NOT BE CHANGED, but the value after the `:` should be changed.  Removed any key word for which you have provided a translation.
+For example:
+
+- if you provided the `name` of the card, and only the `name`, then change to `description, extra`
+- if you provided the `name` of the card and it's `description`, but have not translated the `extra`, then change to `extra`
+- if you provided all 3 entries, then change to `""`
+
+The "untranslated" entry is used during maintanance of the language files to update any remaining default language entries that might have changed.
+
+
+## Special Text
+These character sequences have special meaning in the "description" and "extra" text:
+
+- `<b>` ... `</b>` for **bold**
+- `<i>` ... `</i>` for *italics*
+- `<u>` ... `</u>` for underline
+- `<tab>` and `<t>` and `\t` to add a tab (4 spaces)
+- `<n>` and `\n` for a "hard new line"
+- `<br>` and `<br/>` and `<br />` for a "soft new line"
+- `<c>` and `<center>` add hard new line and center text until the next hard new line
+- `<l>` and `<left>` add hard new line and left align text until the next hard new line
+- `<r>` and `<right>` add hard new line and right align text until the next hard new line
+- `<j>` and `<justify>` add hard new line and justify align text until the next hard new line
+- `<line>` to add a hard new line, a centered dividing line, and a trailing hard new line
+
+Hard new lines (`\n` and `<n>`), will reset the paragraph formatting back to the default.
+Soft new lines will insert a new line, but will continue the current formatting.
+
+The `description` will be default to "center" text.
+
+The `extra` will default to "justify" text.
+
+## Special Images
+Special character sequences are recognized by the program to substitute graphics in the text.  These include:
+
+- `<VP>` and ` VP ` for a Victory Point graphic
+- `Potion` for a small Potion graphic
+- `# Coin`  where # is a number 0 - 13, a question mark `?`, `_`, or the letters `empty` for a small coin graphic with the # on top
+- `# Coins` where # is a number 0 - 13, a question mark `?`, `_`, or the letters `empty` for a small coin graphic with the # on top
+- `# coin`  where # is a number 0 - 13, a question mark `?`, `_`, or the letters `empty` for a small coin graphic with the # on top
+- `# coins` where # is a number 0 - 13, a question mark `?`, `_`, or the letters `empty` for a small coin graphic with the # on top
+- `# Debt` where # is a number 0 - 13, for a small debt graphic with the # on top
+- `# <*COIN*>` where # is a number 0 - 13, will produce a large coin graphic with the # on top
+- `# <*VP*>` where # is a number, will produce a large Victory Point graphic with the # before it
+- `# <*POTION*>` where # is a number, will produce a large Potion graphic with the # before it
+
+For example:
+- the text `1 coin` would produce a graphic of a coin with the number 1 on top.
```

Comment:
It might be good to have some embedded image examples here instead of text. More examples of each special image could be useful too. Fine to leave for a future revision though.